dockerfiles/hpcc/values.yaml

```diff
@@ -71,5 +75,10 @@ notroxie:
     Xconfig: |
       hello there
 
-
-    
\ No newline at end of file
+thor:
+  - name: thor1
+    masterport: 20000
```

Comment:
Why would we want users to tell us these ports?

Reply:
don't know really, and wondered if should remove,

I think the template should specify ports, even though strictly it doesn't have to, to work, because guides suggest the k8s yaml should defined the ports to make it clear which ports the pods use.

Not sure whether I should hardwire to the defaults, or whether I should allow these port options, but make them optional with defaults.